spec/back_end/features/viewing_requests_spec.rb

```diff
@@ -0,0 +1,12 @@
+feature 'viewing requests' do
+  scenario 'a user wants to view their requests' do
+    create_user
+    sign_in
+    list_test_space
```

Comment:
In terms of naming, I think it is implicit that it is a test, so probably I would rename it ```list_space```